snow/engine/snowman/transitive_test.go

```diff
@@ -551,87 +513,41 @@ func TestEngineBlockedIssue(t *testing.T) {
 	require.Equal(blk1.ID(), te.Consensus.Preference())
 }
 
-func TestEngineAbandonResponse(t *testing.T) {
-	require := require.New(t)
-
-	vdr, _, sender, vm, te, gBlk := setupDefaultConfig(t)
-
-	sender.Default(false)
-
-	blk := &snowman.TestBlock{
-		TestDecidable: choices.TestDecidable{
-			IDV:     ids.GenerateTestID(),
-			StatusV: choices.Unknown,
-		},
-		ParentV: gBlk.ID(),
-		HeightV: 1,
-		BytesV:  []byte{1},
-	}
-
-	vm.GetBlockF = func(_ context.Context, blkID ids.ID) (snowman.Block, error) {
-		switch {
-		case blkID == gBlk.ID():
-			return gBlk, nil
-		case blkID == blk.ID():
-			return nil, errUnknownBlock
-		}
-		require.FailNow(errUnknownBlock.Error())
-		return nil, errUnknownBlock
-	}
-
-	require.NoError(te.issue(
-		context.Background(),
-		te.Ctx.NodeID,
-		blk,
-		false,
-		te.metrics.issued.WithLabelValues(unknownSource),
-	))
-	require.NoError(te.QueryFailed(context.Background(), vdr, 1))
-
-	require.Empty(te.blocked)
-}
```

Comment:
This test made 100% no sense. `te.blocked` is always empty during this test... So really it's just testing that `issue` works... Which we already test in other tests.